1-js/06-advanced-functions/12-arrow-functions/article.md

```diff
@@ -112,15 +111,15 @@ function defer(f, ms) {
 }
 ```
 
-Here we had to create additional variables `args` and `ctx` so that the function inside `setTimeout` could take them.
+এখানে আমাদের `setTimeout` ফাংশনের কন্টেক্সট এর জন্য আলাদা দুটি `args` এবং `ctx` ভ্যারিয়েবল লাগছে।
```

Comment:
Please change to: এখানে আমাদের অতিরিক্ত ভেরিয়েবল `args` এবং `ctx` তৈরি করতে হয়েছিল যাতে `setTimeout`-এর ভিতরের ফাংশন সেগুলোকে ব্যবহার পারে।